tket/src/Predicates/PassLibrary.cpp

```diff
@@ -419,4 +419,21 @@ const PassPtr &NormaliseTK2() {
   return pp;
 }
 
+const PassPtr &ZZPhaseToRz() {
+  static const PassPtr pp([]() {
+    Transform t = Transforms::ZZPhase_to_Rz();
+    // GateSetPredicate not preserved because ZZPhase gates may be converted to
+    // Rz gates
+    PredicateClassGuarantees g_postcons = {
+        {typeid(GateSetPredicate), Guarantee::Clear}};
+    PostConditions postcon = {{}, g_postcons, Guarantee::Preserve};
+    PredicatePtrMap precons;
+    // record pass config
+    nlohmann::json j;
+    j["name"] = "ZZPhaseToRz";
```

Comment:
Have you made a testcase checking this in the json generation?

Reply:
Though as its a standard pass with no special arguments do we need one? It seems that not all passes do have one

Reply:
Maybe we don't need one in that case, thought there are testcases for all passes.

Reply:
and tbf i forgot to add it to compiler_pass_v1.json so probably a good reason to have the tests ha

Reply:
added tests + fixed